sumpy/fmm.py

```diff
@@ -548,7 +548,7 @@ def box_target_list_kwargs(self):
     # }}}
 
     def run_opencl_fft(self, queue, input_vec, inverse, wait_for, inplace):
-        app = self.tree_indep.opencl_fft_app(input_vec.shape, input_vec.dtype,
+        app = get_opencl_fft_app(queue, input_vec.shape, input_vec.dtype,
```

Comment:
How come you decided to remove caching outright?

Reply:
Debugging on CI to see if removing the cache fixes everything. It does fix most of the tests which means different queues for configuration and launch in vkfft doesn't work.

Reply:
Ugh, super annoying. :/

Reply:
Nvm, that was my bad. Fixing https://github.com/vincefn/pyvkfft/pull/17 should allow us to use different queues. I've made the change here, but we have to wait for https://github.com/vincefn/pyvkfft/pull/17 to be accepted and released in conda package.